server/schedule/operator/create_operator.go

```diff
@@ -217,7 +218,7 @@ func CreateScatterRegionOperator(desc string, ci ClusterInformer, origin *core.R
 		EnableLightWeight().
 		// EnableForceTargetLeader in order to ignore the leader schedule limit
 		EnableForceTargetLeader().
-		Build(0)
+		Build(OpAdmin)
```

Comment:
maybe expose one param to set OpKind is better?

Reply:
We can regard all scatter operator as admin type in the current scenarios.